### PR TITLE
Promote `google_essential_contacts_contact` to GA

### PR DESCRIPTION
**Pull request by upodroid**
Fixes: https://github.com/hashicorp/terraform-provider-google/issues/9783

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
essentialcontacts: promoted `google_essential_contacts_contact` to GA
```

### CODE REVIEW

**Comment by upodroid** [comment]
@c2thorn 

The import_format is a bit tricky.

```
2021/08/16 19:43:44 [DEBUG] matching ID projects/REDACTED/contacts/4 to regex (?P<parent>[^/]+)/contacts.
2021/08/16 19:43:44 [DEBUG] importing parent = REDACTED
2021/08/16 19:43:44 [WARN] Got error running Terraform: exit status 1

Error: nil entry in ImportState results. This is always a bug with
the resource that is being imported. Please report this as
a bug to Terraform.


    provider_test.go:270: Step 2/2 error running import: exit status 1
        
        Error: nil entry in ImportState results. This is always a bug with
        the resource that is being imported. Please report this as
        a bug to Terraform.
        
```

import format should be {{parent}}/contacts/{{contact_id}} except contact_id doesn't exist and {{parent}} is various selflinks for orgs,folders and projects :(

**Comment by ScottSuarez** [comment]
Try doing [this](https://github.com/GoogleCloudPlatform/magic-modules/pull/5010/files) instead of adding contactId.

It should work.

**Comment by upodroid** [comment]
thanks, it worked

File: mmv1/products/essentialcontacts/api.yaml [hunk 1]
```diff
@@ -54,6 +56,11 @@ objects:
         output: true
         description: |
           The identifier for the contact. Format: {resourceType}/{resource_id}/contacts/{contact_id}
+      - !ruby/object:Api::Type::String
+        name: 'contactId'
+        output: true
+        description: |
+          Contact Id value
```
Comment (ScottSuarez) [inline mmv1/products/essentialcontacts/api.yaml:63]:
```suggestion
```

Any value from adding this field?

File: mmv1/products/essentialcontacts/api.yaml [hunk 2]
```diff
@@ -18,6 +18,9 @@ versions:
   - !ruby/object:Api::Product::Version
     name: beta
     base_url: https://essentialcontacts.googleapis.com/v1beta1/
```
Comment (ScottSuarez) [inline mmv1/products/essentialcontacts/api.yaml:20]: if beta is deprecated should we delete beta endpoint? 

The beta provider will switch to use the ga enpoint.

Comment (upodroid) [reply mmv1/products/essentialcontacts/api.yaml]: Yeah, fixed

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**